lib/scenic/adapters/postgres.rb

```diff
@@ -77,6 +77,14 @@ def refresh_materialized_view(name)
         execute "REFRESH MATERIALIZED VIEW #{name};"
       end
 
+      # Refreshes a materialized view concurrently from its SQL schema.
+      #
+      # @param name The name of the materialized view to refresh..
+      # @return [void]
+      def refresh_materialized_view_concurrently(name)
```

Comment:
If we used the earlier suggestion, this method would go away, and `refresh_materialized_view` would change its signature to `def refresh_materialized_view(name, concurently: false, **args)`.